Add release note for 2201.5.5 (#8530)
ballerina-platform/ballerina-dev-website: downloads/swan-lake-release-notes/swan-lake-2201.5.5/RELEASE_NOTE.md

---
layout: ballerina-left-nav-release-notes
title: Swan Lake Update 5 (2201.5.5)
permalink: /downloads/swan-lake-release-notes/2201.5.5/
active: 2201.5.5
redirect_from:
- /downloads/swan-lake-release-notes/2201.5.5
- /downloads/swan-lake-release-notes/2201.5.5/
- /downloads/swan-lake-release-notes/2201.5.5-swan-lake/
- /downloads/swan-lake-release-notes/2201.5.5-swan-lake
---

## Overview of Ballerina Swan Lake Update 5 (2201.5.5)

> <em>Swan Lake 2201.5.5 is the fifth patch release of Ballerina 2201.5.0 (Swan Lake Update 5) and it includes a new set of bug fixes to the tooling.</em>

Reviewer: ```suggestion
<em>Swan Lake 2201.5.5 is the fifth patch release of Ballerina 2201.5.0 (Swan Lake Update 5) and it includes a new set of bug fixes to developer tools.</em>
```

---
layout: ballerina-left-nav-release-notes
title: Swan Lake Update 5 (2201.5.5)
permalink: /downloads/swan-lake-release-notes/2201.5.5/
active: 2201.5.5
redirect_from:
- /downloads/swan-lake-release-notes/2201.5.5
- /downloads/swan-lake-release-notes/2201.5.5/
- /downloads/swan-lake-release-notes/2201.5.5-swan-lake/
- /downloads/swan-lake-release-notes/2201.5.5-swan-lake
---

## Overview of Ballerina Swan Lake Update 5 (2201.5.5)

> <em>Swan Lake 2201.5.5 is the fifth patch release of Ballerina 2201.5.0 (Swan Lake Update 5) and it includes a new set of bug fixes to the <ADD_THE_LIST_OF_COMPONENTS>.</em>

Reviewer: ```suggestion
<em>Swan Lake 2201.5.5 is the fifth patch release of Ballerina 2201.5.0 (Swan Lake Update 5) and it includes a new set of bug fixes to the <ADD_THE_LIST_OF_COMPONENTS>.</em>
```
Need to fill the `<ADD_THE_LIST_OF_COMPONENTS>`.

## Update Ballerina

Update your current Ballerina installation directly to 2201.5.5 by using the [Ballerina Update Tool](/learn/update-tool/) as follows.

1. Run `bal update` to get the latest version of the Update Tool.
2. Run `bal dist pull 2201.5.5` to update to this latest distribution.

## Install Ballerina

If you have not installed Ballerina, then, download the [installers](/downloads/#swanlake) to install.

> <!-- ADD ONLY THE APPLICABLE SECTIONS FROM THE BELOW -->

Reviewer: ```suggestion
<!-- ADD ONLY THE APPLICABLE SECTIONS FROM THE BELOW -->
```
Add only the sections that are applicable for this release from the template.

## Language updates

### Bug fixes

To view bug fixes, see the [GitHub milestone for Swan Lake Update 8 (<RELEASE_VERSION>)](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+label%3AType%2FBug+is%3Aclosed+milestone%3A<RELEASE_VERSION>).

## Runtime updates

### Bug fixes

To view bug fixes, see the [GitHub milestone for Swan Lake Update 8 (<RELEASE_VERSION>)](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A<RELEASE_VERSION>+label%3ATeam%2FjBallerina+label%3AType%2FBug+is%3Aclosed).

## Ballerina library updates

### Bug fixes

To view bug fixes, see the [GitHub milestone for Swan Lake Update 8 (<RELEASE_VERSION>)](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aissue+label%3AType%2FBug+is%3Aclosed+milestone%3A<RELEASE_VERSION>).

## Developer tools updates

### Bug fixes

To view bug fixes, see the GitHub milestone for Swan Lake Update 8 (<RELEASE_VERSION>) of the repositories below.

- [Language server](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+label%3ATeam%2FLanguageServer+milestone%3A<RELEASE_VERSION>+is%3Aclosed+label%3AType%2FBug+)

> - [OpenAPI](https://github.com/ballerina-platform/openapi-tools/issues?q=is%3Aissue+label%3AType%2FBug+milestone%3A%22Swan+Lake+<RELEASE_VERSION>%22+is%3Aclosed)

Reviewer 2: ```suggestion
- [OpenAPI](https://github.com/ballerina-platform/ballerina-library/issues?q=is%3Aissue+label%3Amodule%2Fopenapi-tools+milestone%3A2201.5.5+is%3Aclosed)
```
